### ISSUE TITLE
Product Owner vs Product Manager

### ISSUE THREAD

**Issue by eharrisSSA** [usds/playbook#187]
For play #6, is it possible to be less direct about the Product Owner?  A lot of times, agencies focus heavily on the role and less on the functions needed to be performed.  Aren't we more concerned with having an accountable authoritative person  responsible for the "success or failure of the overall service?"

I ask because these duties can be performed by a Product Manager.  We have a Product Manager looking outward strategically and a Product Owner looking inward tactically.  But saying product owner, and not listing alternatives or allowing for flexibility, makes it hard to implement when Sr. Leaders look to guidebooks for guidance.  

Is it possible to say "Product Owner or Manager" showing that different agile process may call this role by different names, but in the end, the functions are largely the same.  

Thanks for the consideration.

**Comment by rdymond1** [comment]
The Product Manager and Product Owner roles largely overlap. In the large
majority of cases I can't see a need to have both roles. Having both would
lead to waste, including handoffs, information loss, over production of
backlog items, and diffused responsibilities.

The Product Owner is responsible for delivering a return on investment ROI
for the cost and time invested. Having a product manager blocking their
access to customers and assuming responsibility for the customer
relationship without being responsible for delivering the value is a recipe
for long term systemic problems.

There are a few exceptions to this pattern. For example if the organization
has a large customer base that is fragmented and 500+ team members on 50 to
100 teams. In large complex solutions product owners and product managers
may coexist due to the size and complexity of the system and
customer/stakeholder base. In this case a scaling framework such as Large
Scale Scrum (LeSS) provides a good solution.

Robin Dymond, CST
Managing Partner, Innovel
www.innovel.net
www.scrumtraining.com
Direct: (804) 239-4329
twitter: @robindymond
Linkedin: https://www.linkedin.com/in/robindymond

For play #6 <https://github.com/usds/playbook/pull/6>, is it possible to be
less direct about the Product Owner? A lot of times, agencies focus heavily
on the role and less on the functions needed to be performed. Aren't we
more concerned with having an accountable authoritative person responsible
for the "success or failure of the overall service?"

I ask because these duties can be performed by a Product Manager. We have a
Product Manager looking outward strategically and a Product Owner looking
inward tactically. But saying product owner, and not listing alternatives
or allowing for flexibility, makes it hard to implement when Sr. Leaders
look to guidebooks for guidance.

Is it possible to say "Product Owner or Manager" showing that different
agile process may call this role by different names, but in the end, the
functions are largely the same.

Thanks for the consideration.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/usds/playbook/issues/187>, or mute the thread
<https://github.com/notifications/unsubscribe/AIEwW3gDRXa4-SR95Y1AIzuLsPwzK0HWks5qTWJdgaJpZM4JHeJz>
.

**Comment by cew821** [comment]
Hi @eharrisSSA and @rdymond1, thanks for the feedback. For the purposes of this section, I think we are using "product owner" and "product manager" interchangeably. We chose "product owner" since that term tended to be more common in government, while "product manager" is a term more common in the private sector.

We agree with the sentiments you are expressing. The important thing is that there is an individual responsible for overall success of the product... regardless of the title that person holds. I think we ware open to a change that makes this more clear. Is there a specific wording change you could recommend?

**Comment by eharrisSSA** [comment]
@cew821 I agree.  I'm more interested in the function/responsibility verses the title.  However, gaining buy-in from executives who look at guidance like this playbook, see the term "owner" and make it harder to navigate the various possibilities for improvement (like introducing core-assets for product lines).  

I would recommend simply saying manager/owner, or stating the responsibility and making the title an parenthetical example (i.e. product owner or product manager) would take care of it. 

Thank you for your attention.  This is GREAT and much needed.
